safe-global/safe-wallet-web
# fix: don't show banner if preferences exist

## What it solves

Resolves https://github.com/safe-global/safe-wallet-web/pull/2592#issuecomment-1749912718

## How this PR fixes it

The notification registration banner now takes into account whether the current Safe is registered before being displayed.

## How to test it

1. Ensure a Safe is added and _not_ registered for notifications.
2. Open the Safe in question but don't dismiss the notification registration banner.
3. Open the notification settings for that Safe.
4. Register for notifications.
5. Observe the banner close.
6. Reopening that Safe should not show the banner until it is "unregistered" for notifications.

## Screenshots

![notification banner](https://github.com/safe-global/safe-wallet-web/assets/20442784/3b5c1d7b-4e53-4eca-b032-ca819071ec6f)

## Checklist
